js/main.js

```diff
@@ -40,6 +48,37 @@ Site = {
 
 };
 
+Site.Shop = {
+  init: function() {
+    var _this = this;
+
+    this.Search.init();
+  },
+
+  Search: {
+    init: function() {
+      var _this = this;
+
+      if ($('#searchform').length) {
+        _this.bindFocus();
+      }
+    },
+
+    bindFocus: function() {
```

Comment:
the search looks like this: 

![screen shot 2016-08-12 at 17 46 24](https://cloud.githubusercontent.com/assets/845186/17639081/b7afe34e-60b4-11e6-9273-20b5467e94db.png)

There is no icon or toggling interaction. And maybe it would be better not to override the submit but just have a toggle visible first and then remove and put the form in to replace it